pandas/tests/frame/test_convert_to.py

```diff
@@ -577,7 +578,7 @@ def test_frame_to_dict_tz(self):
             (
                 defaultdict(list),
                 defaultdict(
-                    list,
+                    Callable[[], dict],
```

Comment:
Not sure this is the correct change; the default factory argument probably shouldn't change. What was the error you are trying to fix?

Reply:
pandas\tests\frame\test_convert_to.py:579: error: Argument 1 to "defaultdict" has incompatible type "Type[List[Any]]"; expected "Optional[Callable[[], Dict[str, float]]]"

Reply:
I think you need to annotate the defaultdict not change it. Might have some luck looking through https://github.com/python/mypy/issues/995

Reply:
Thanks for the link.  I'll give that a try this evening.

Reply:
I've tried different annotations with no luck (other than "#type: ignore" obviously), it just keeps asking for the Callable type.  Any other suggestions?

Reply:
Can you use typing.DefaultDict? Haven't tried but just need to do this somehow without changing implementation I think

Reply:
I tried using DefaultDict, but it doesn't seem like it recognizes the constructor with the variadic data arguments.  I could keep looking.

Reply:
Hmm I suppose could also change the factory to a `dict` as that is what the values of the object actually are. Not sure why `list` was chosen

Reply:
@WillAyd That's exactly what I just realized and came here to mention.  Because the defaultdict has dictionary values, it should return an empty dictionary if the key doesn't exist, not a list.  Was trying so hard to get the annotation to work that I didn't notice that.

Reply:
Once I changed the argument to a dict, no annotations were necessary.  Looks like this was an argument bug, not an annotation issue.